app/safe/internal/server/handle/handle.go

```diff
@@ -58,6 +58,15 @@ func InitializeRoutes() {
 			return
 		}
 
+		// Route to delete secrets from Aegis Safe.
```

Comment:
I need to implement this in Aegis Sentinel; and also modify the documentation too.